tests/postgres_tests/test_aggregates.py

```diff
@@ -128,6 +128,24 @@ def test_json_agg_empty(self):
         self.assertEqual(values, json.loads('{"jsonagg": []}'))
 
 
+class TestArrayAggDistinct(PostgreSQLTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        AggregateTestModel.objects.create(char_field='Foo')
+        AggregateTestModel.objects.create(char_field='Foo')
+        AggregateTestModel.objects.create(char_field='Bar')
+
+    def test_string_agg_distinct_false(self):
+        values = AggregateTestModel.objects.aggregate(arrayagg=ArrayAgg('char_field', distinct=False))
+        agg = list(sorted(values['arrayagg']))
+        self.assertEqual(agg, ['Bar', 'Foo', 'Foo'])
```

Comment:
I don't think `list()` is needed. `self.assertEqual(sorted(values['arrayagg'], ['Bar', 'Foo', 'Foo'])` looks good to me.